includes/plugins/wc-memberships/class-memberships.php

```diff
@@ -334,13 +334,8 @@ private static function get_block_names_recursive( $blocks ) {
 	 * }
 	 */
 	public static function get_gate_metadata() {
```

Comment:
I believe this is also used by back-end events... maybe we need to keep things here and override on the front-end... but for the back-end events we would actually need to see how that conditional blocks are rendered to find out what values to set

Reply:
Ah, I didn't see that the back-end events were actually using this. Maybe we can avoid recreating the Memberships blocks' logic by checking their render output.

Reply:
2f117daa adds a check in the back-end method, too. We don't get innerBlocks if a member or non-member block won't render any content.